internal/engine/buildcontroller.go

```diff
@@ -212,14 +214,13 @@ func (c *BuildController) OnChange(ctx context.Context, st store.RStore) {
 		}
 		ctx := logger.WithLogger(ctx, logger.NewLogger(logger.Get(ctx).Level(), actionWriter))
 
-		filesChanged := entry.buildStateSet.FilesChanged()
```

Comment:
pretty sure `bss.FilesChanged` is obsolete now

Reply:
it's used up in `needsBuild` now: https://github.com/windmilleng/tilt/pull/2042/files#diff-fe6c9f90c8a07f06670c56f531886cedR192